.travis.yml

```diff
@@ -10,8 +10,8 @@ install:
 before_script:
   - flake8 . --count --max-complexity=15 --show-source --statistics
 script:
-  - python app/server.py > /dev/null &
-  - pytest --capture=sys --showlocals
```

Comment:
Why remove __—showlocals__?  I find that option quite helpful.

Reply:
I will add the `showlocals` option. I guess, I overlooked this while fixing conflicts, apologies for that